### PR TITLE
Add SetFit installation instructions for T-Few baseline

### PR DESCRIPTION
**Pull request by danielkorat**
Fixes issue with T-Few setup which was unable to import util classes from SetFit package.

### CODE REVIEW

**Comment by tomaarsen** [comment]
The `.[dev]` install has quite a few dependencies nowadays:
https://github.com/huggingface/setfit/blob/0cb8ffdb59f896709a565a4c01bbc4636e7e3b39/setup.py#L12-L33

Do you think we could get away with just using `python -m pip install -e .`? I believe that neither `Benchmark` nor the `load_data_splits` methods have weird dependencies. It might save someone some disk space.

Other than that, I'm quite satisfied with this solution.

- Tom Aarsen

**Comment by danielkorat** [comment]
@tomaarsen That might work, but I don't have bandwidth to test it right now :(

**Comment by tomaarsen** [comment]
I'll have more time to dig into it myself starting tomorrow.

**Comment by danielkorat** [comment]
Thank you 🙏
________________________________

**Comment by tomaarsen** [comment]
I've tested it out with a fresh install. `python -m pip install -e .` is sufficient to access `Benchmark` and `load_data_splits`.

File: scripts/tfew/README.md [hunk 1]
```diff
@@ -4,10 +4,11 @@ These scripts run the baselines based on the `T-Few` paper: [_Few-Shot Parameter
 
 ## Setup
 
-To run the scripts, first create a Python virtual environment, e.g. with `conda`:
+To run the scripts, first create a Python virtual environment, e.g. with `conda`, and install SetFit from source:
 
 ```
 conda create -n baselines-tfew python=3.7 && conda activate baselines-tfew
+python -m pip install -e '.[dev]'
```
Comment (tomaarsen) [inline scripts/tfew/README.md:11]:
```suggestion
python -m pip install -e .
```
This should work fine, too.

**Comment by danielkorat** [comment]
Great, can you merge please?
________________________________

**Comment by tomaarsen** [comment]
There's two solutions at this stage:
1. We merge this right now, or
2. We add `setfit` to the [T-few requirements.txt](https://github.com/SetFit/t-few/blob/master/requirements.txt), and we close this PR as it would no longer be needed.

I think the second might be better. What do you think?

**Comment by danielkorat** [comment]
I agree
________________________________

**Comment by tomaarsen** [comment]
Superseded by https://github.com/SetFit/t-few/pull/2.

**Pull request closed without merging**